diff -Nru libsndfile-0.0.21/configure.in libsndfile-0.0.21.new/configure.in
--- libsndfile-0.0.21/configure.in	Tue Aug 15 22:44:47 2000
+++ libsndfile-0.0.21.new/configure.in	Mon Aug 21 02:21:51 2000
@@ -44,12 +44,6 @@
 AC_ARG_ENABLE(justsrc, [  --enable-justsrc        only compile library sources ],
 	ac_arg_justsrc="$enableval", ac_arg_justsrc="N")
 
-if test $ac_cv_prog_gcc = yes; then
-	CFLAGS="-g -O2 -Wall -pedantic -Wstrict-prototypes -pipe"
-	dnl CFLAGS="-g -O0 -Wall -Werror -Wstrict-prototypes -pipe"
-	dnl AC_MSG_WARN([**** Optimisation has been switched off. ****])
-	fi
-
 if test $cross_compiling = yes ; then
 	AC_MSG_WARN([******************************************************************])
 	AC_MSG_WARN([*** We are cross-compiling, so have to assume sizeof (short) == 2 ])
