--- libsndfile-0.0.23/configure.in.orig	Sun Jul 15 11:10:03 2001
+++ libsndfile-0.0.23/configure.in	Thu Jul 19 23:25:23 2001
@@ -54,26 +54,6 @@
 						[  --enable-force-broken-float  force use of broken float code ],
 	ac_arg_broken_float="Y", ac_arg_broken_float="N")
 
-if test $ac_cv_prog_gcc = yes ; then
-	CFLAGS="-g -O2 -Wstrict-prototypes"
-	if test "$ac_arg_gcc_opt" = "N" ; then
-		CFLAGS="-g -O0 -Wstrict-prototypes"
-		AC_MSG_WARN([[*** Compiler optimisations switched off. ***]])
-		fi
-	dnl Disable -Wall and -pedantic for Apple Darwin/Rhapsody.
-	dnl System headers on these systems are broken.
-	case "$target_os" in 
-		darwin* | rhapsody*)
-			;;
-		*)
-			CFLAGS="$CFLAGS -Wall -pedantic"
-			;;
-		esac
-	if test $ac_arg_gcc_pipe != "N" ; then
-	 	CFLAGS="$CFLAGS -pipe"
-	 	fi
-	fi
-	
 if test $cross_compiling = yes ; then
 	AC_MSG_WARN([[******************************************************************]])
 	AC_MSG_WARN([[*** We are cross-compiling, so have to assume sizeof (short) == 2 ]])
