--- libsndfile-0.0.25/configure.in.orig	Sun Sep 23 22:58:31 2001
+++ libsndfile-0.0.25/configure.in	Wed Sep 26 06:24:46 2001
@@ -47,33 +47,10 @@
 AC_ARG_ENABLE(gcc-pipe, [  --disable-gcc-pipe           disable gcc -pipe option ],
 	ac_arg_gcc_pipe="N", ac_arg_gcc_pipe="Y")
 
-AC_ARG_ENABLE(gcc-opt,  [  --disable-gcc-opt            disable gcc optimisations ],
-	ac_arg_gcc_opt="N", ac_arg_gcc_opt="Y")
-
 AC_ARG_ENABLE(force-broken-float,  
 						[  --enable-force-broken-float  force use of broken float code ],
 	ac_arg_broken_float="Y", ac_arg_broken_float="N")
 
-if test $ac_cv_prog_gcc = yes ; then
-	CFLAGS="$CFLAGS -g -O2 -Wstrict-prototypes"
-	if test "$ac_arg_gcc_opt" = "N" ; then
-		CFLAGS="$CFLAGS -g -O0 -Wstrict-prototypes"
-		AC_MSG_WARN([[*** Compiler optimisations switched off. ***]])
-		fi
-	dnl Disable -Wall and -pedantic for Apple Darwin/Rhapsody.
-	dnl System headers on these systems are broken.
-	case "$target_os" in 
-		darwin* | rhapsody*)
-			;;
-		*)
-			CFLAGS="$CFLAGS -Wall -pedantic"
-			;;
-		esac
-	if test $ac_arg_gcc_pipe != "N" ; then
-	 	CFLAGS="$CFLAGS -pipe"
-	 	fi
-	fi
-	
 if test $cross_compiling = yes ; then
 	AC_MSG_WARN([[******************************************************************]])
 	AC_MSG_WARN([[*** We are cross-compiling, so have to assume sizeof (short) == 2 ]])
