--- libsndfile-1.0.19/tests/vorbis_test.c.orig	2009-02-22 00:06:45.000000000 +0100
+++ libsndfile-1.0.19/tests/vorbis_test.c	2009-03-13 15:22:03.062762569 +0100
@@ -96,8 +96,8 @@
 	for (k = 0 ; k < ARRAY_LEN (float_data) ; k ++)
 		max_abs = max_float (max_abs, fabs (float_data [k])) ;
 
-	if (max_abs > 1.02)
-	{	printf ("\n\n    Error : max_abs %f should be < 1.02.\n\n", max_abs) ;
+	if (max_abs > 1.021)
+	{	printf ("\n\n    Error : max_abs %f should be < 1.021.\n\n", max_abs) ;
 		exit (1) ;
 		} ;
 
