--- libsndfile-1.0.25/configure.ac.orig	2011-07-13 10:59:57.000000000 +0200
+++ libsndfile-1.0.25/configure.ac	2015-03-06 17:18:22.620080669 +0100
@@ -335,7 +335,7 @@
 		enable_external_libs=yes
 
 		EXTERNAL_CFLAGS="$FLAC_CFLAGS $OGG_CFLAGS $VORBISENC_CFLAGS $SPEEX_CFLAGS"
-		EXTERNAL_LIBS="$FLAC_LIBS $VORBISENC_LIBS $SPEEX_LIBS"
+		EXTERNAL_LIBS="$FLAC_LIBS $VORBISENC_LIBS $VORBIS_LIBS $SPEEX_LIBS $OGG_LIBS"
 	else
 		echo
 		AC_MSG_WARN([[*** One or more of the external libraries (ie libflac, libogg and]])
