diff -Nru libsndfile-0.0.18/configure.in libsndfile-0.0.18.new/configure.in
--- libsndfile-0.0.18/configure.in	Tue Apr 25 08:04:11 2000
+++ libsndfile-0.0.18.new/configure.in	Sun May 14 23:14:29 2000
@@ -40,12 +40,6 @@
 AC_ARG_ENABLE(justsrc, [  --enable-justsrc        only compile library sources ],
 	ac_arg_justsrc="$enableval", ac_arg_justsrc="N")
 
-if test $ac_cv_prog_gcc = yes; then
-	CFLAGS="-g -O2 -Wall -pedantic -Wstrict-prototypes -pipe"
-	dnl CFLAGS="-g -O0 -Wall -pedantic-errors -Wstrict-prototypes -pipe"
-	dnl AC_MSG_WARN([**** Optimisation has been switched off. ****])
-	fi
-
 if test $cross_compiling = yes ; then
 	AC_MSG_WARN([******************************************************************])
 	AC_MSG_WARN([*** We are cross-compiling, so have to assume sizeof (short) == 2 ])
