--- libsndfile-1.0.26/M4/octave.m4.orig	2015-11-25 21:57:25.675180371 +0100
+++ libsndfile-1.0.26/M4/octave.m4	2015-11-26 15:46:34.252488392 +0100
@@ -119,7 +119,7 @@
 			2.*)
 				AC_MSG_WARN([Octave version 2.X is not supported.])
 				;;
-			3.*)
+			3.*|4.*)
 				OCTAVE_DEST_ODIR=`$OCTAVE_CONFIG --oct-site-dir | sed 's%^/usr%${prefix}%'`
 				OCTAVE_DEST_MDIR=`$OCTAVE_CONFIG --m-site-dir | sed 's%^/usr%${prefix}%'`
 
