--- libsndfile-1.0.20/M4/octave.m4.orig	2009-10-25 15:51:00.461732576 +0100
+++ libsndfile-1.0.20/M4/octave.m4	2009-10-25 15:51:32.459533439 +0100
@@ -119,7 +119,7 @@
 			2.*)
 				AC_MSG_WARN([Octave version 2.X is not supported.])
 				;;
-			3.0.*)
+			3.*|4.*)
 				OCTAVE_DEST_ODIR=`$OCTAVE_CONFIG --oct-site-dir | sed 's%^/usr%${prefix}%'`
 				OCTAVE_DEST_MDIR=`$OCTAVE_CONFIG --m-site-dir | sed 's%^/usr%${prefix}%'`
 
