--- libsndfile-1.2.2/Octave/Makefile.am.orig	2022-11-24 15:04:49.000000000 +0100
+++ libsndfile-1.2.2/Octave/Makefile.am	2025-02-14 17:56:28.054195607 +0100
@@ -21,7 +21,7 @@
 OCT_LIBS = @OCT_LIBS@
 
 SNDFILEDIR = $(top_builddir)/src
-AM_CPPFLAGS = -I$(SNDFILEDIR)
+AM_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/src
 
 oct_module_srcs = sndfile.cc
 oct_module_files = sndfile.oct PKG_ADD
@@ -39,10 +39,10 @@
 # Use Octave's mkoctfile to do all the heavy lifting. Unfortunately, its
 # a little dumb so we need to guide it carefully.
 sndfile.oct : sndfile.o
-	$(QUIET_GEN) $(MKOCTFILE) -v $(INCLUDES) $(top_builddir)/Octave/$+ -L$(SNDFILEDIR)/.libs -L$(SNDFILEDIR) -lsndfile -o $(top_builddir)/Octave/$@ > /dev/null
+	$(QUIET_GEN) $(MKOCTFILE) -v $(AM_CPPFLAGS) $(top_builddir)/Octave/$+ -L$(SNDFILEDIR)/.libs -L$(SNDFILEDIR) -lsndfile -o $(top_builddir)/Octave/$@ > /dev/null
 
 sndfile.o : sndfile.cc
-	$(QUIET_GEN) $(MKOCTFILE) -v $(INCLUDES) -c $+ -o $(top_builddir)/Octave/$@ > /dev/null
+	$(QUIET_GEN) $(MKOCTFILE) -v $(AM_CPPFLAGS) -c $+ -o $(top_builddir)/Octave/$@ > /dev/null
 
 # Allow for the test being run in the build dir, but the test script
 # being located in the source dir.
